docs/orchestrate/getting-started.md

```diff
@@ -11,6 +11,8 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
 This page outlines how to run your browser and mobile tests in Sauce Orchestrate.
```

Comment:
If SO is for enterprise plans only, we should also add a badge at the top of the page to clarify.
```suggestion
<p><span className="sauceGreen">Enterprise Plans only</span></p>

This page outlines how to run your browser and mobile tests in Sauce Orchestrate.
```

Reply:
@lysannep this isn't ONLY for enterprise, it's a paid add-on all customers can access